src/labone/nodetree/demo.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "initial_id",
+   "metadata": {
+    "collapsed": true,
+    "ExecuteTime": {
+     "end_time": "2023-10-12T16:43:41.403568275Z",
+     "start_time": "2023-10-12T16:43:41.191171796Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "kj/filesystem-disk-unix.c++:1734: warning: PWD environment variable doesn't match current directory; pwd = /home/rayk/PycharmProjects/labone-python\n"
+     ]
+    }
+   ],
+   "source": [
+    "from labone.nodetree.node import *\n",
+    "import asyncio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "outputs": [],
+   "source": [
+    "session = await KernelSession.create(\n",
+    "        kernel_info=ZIKernelInfo(),\n",
+    "        server_info=ServerInfo(host='localhost', port=8004)\n",
+    "    )\n",
+    "zi = await get_nodetree(session)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-10-12T16:43:56.028425167Z",
+     "start_time": "2023-10-12T16:43:55.991793662Z"
+    }
+   },
+   "id": "b7c4b4067f43ca28"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "AnnotatedValue(value=</zi/debug/level.status: 3>, path='/zi/debug/level', timestamp=1697128485, extra_header=None)"
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "await zi.debug.level()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-10-12T16:52:21.886532323Z",
+     "start_time": "2023-10-12T16:52:21.866066523Z"
+    }
+   },
+   "id": "b89e4b6cf0f2cc80"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "AnnotatedValue(value=</zi/debug/level.error: 5>, path='/zi/debug/level', timestamp=1697129571, extra_header=None)"
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "await zi.debug.level(5)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-10-12T16:52:51.573006952Z",
+     "start_time": "2023-10-12T16:52:51.551423988Z"
+    }
+   },
+   "id": "19055d9aa79b6ff5"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "AnnotatedValue(value=0, path='/zi/mds/groups/0/status', timestamp=1697128485, extra_header=None)"
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result = await zi.mds.groups[0]()\n",
```

Comment:
Change the __repr__ to the following format

`Result for <nodepath> at <timestamp>`

e.g.

`Results for /zi/mds/groups/0 at 1697183747`